code/datums/supplypacks.dm

```diff
@@ -1027,9 +1027,9 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 					/obj/item/weapon/gun/energy/smalldisintegrator,
 					/obj/item/weapon/gun/energy/smalldisintegrator)
 	cost = 60
-	containertype = /obj/structure/closet/crate/secure/weapon
+	containertype = /obj/structure/closet/crate/secure/ayyweapon
 	containername = "MDF standard weapons crate"
-	access = list(access_armory)
+	one_access = list(access_mothership_military)
```

Comment:
the way it should look is this:
`one_access = list(access_armory, access_mothership_military)`

This tells the game that *either* of these two accesses will open it

Reply:
ah, okay